### DISCUSSION TITLE
Relative path - building local documentation with no Web server

### DISCUSSION THREAD

**Discussion by goozoon** [squidfunk/mkdocs-material#5238] [General]
Is it possible to build a documentation for local use with no Web Server in the back?
Like build and start index.html from usb stock or cd...

**Comment by squidfunk** [comment]
Yes, please read our documentation [on offline usage](https://squidfunk.github.io/mkdocs-material/setup/building-for-offline-usage/). Many users and companies are already shipping their documentation offline to their users. Search will also work offline, if you use the [`offline` plugin](https://squidfunk.github.io/mkdocs-material/setup/building-for-offline-usage/#built-in-offline-plugin). There are [some limitations](https://squidfunk.github.io/mkdocs-material/setup/building-for-offline-usage/#limitations).
